### ISSUE TITLE
Missing config file and documentation for it

### ISSUE THREAD

**Issue by RobinSattler** [smash-transport/smash-hadron-sampler#22]
I recently installed the [smash-vhlle-hybrid](https://github.com/smash-transport/smash-vhlle-hybrid) and while setting up the hadron sampler I noticed that no config file is shipped with it. Although the hybrid handler does contain a config file for it, it would be great to include a config file in the sampler as well since users might not use the hybrid handler. 

Also a short documentation or rather a simple explanation about the config file of the sampler and the possible/needed inputs would be nice! :slightly_smiling_face:

**Comment by RobinSattler** [comment]
I started tackling this issue and noticed that there are parameters `NBINS`, `QMAX`, `rescatter`, and `weakContribution` that are initialized, but never used anywhere in the code. @yukarpenko, is there a reason for this (or did I maybe overlook something)? If there's no reason can they safely be removed?

**Comment by yukarpenko** [comment]
I think all of these parameters can be (and should be, actually) removed. A long, long time ago, in a distant galaxy, the code was a part of a larger framework which was also computing Cooper-Frye integrals via direct numerical integration - the parameters `NBINS` and `QMAX` trail from that era. Later on, when I was using UrQMD as a final-state hadronic cascade, I was compiling it together with the hydro code - hence there are `rescatter` and `weakContribution` parameters. 
All of that has been removed long time ago but it seems that the removal was clumsy - the parameters are still there.

**Comment by RobinSattler** [comment]
@yukarpenko, thanks for your answer. I opened a pull request to fix all these issues.

**Comment by RobinSattler** [comment]
I had a look at the commit. This would be an option. "events" mode input is not required anymore (what was it good for in the first place)?

I guess most of the SMASH group uses the hybrid handler for multiple stage calculations and hence this is no improvement for us in that sense. Additionally, the hybrid handler framework would have to be adjusted as well. @NGoetz, what are you thinking about this idea?

**Comment by yukarpenko** [comment]
Nope, the key for the events mode is redundant now, since I believe (without looking up the current code though) it is the only mode the sampler works in now. A long, long time ago, in a distant galaxy, there was a mode to estimate the maximal value of the distribution function(s) that would be later on used for sampling - before I figured an approximate analytical expression for it.
I'll try (again) the hybrid handler. In practice in my simulations I use a 50-line script (including inter-block spaces) to run it all...

**Comment by NGoetz** [comment]
@RobinSattler although it is true that it would be no improvement for us, we would not have to change the handler as long as the changes are optional in some sense. Though I think cleaning up the current state, especially the thing with "event", is worth the small change on the handler side of things.

**Comment by RobinSattler** [comment]
@NGoetz, alright, and if there is a change needed, I can implement it in my upcoming change in the hybrid. Should be only a small change.

@yukarpenko, I'll open a new issue for this and close this one.
